src/schema/metadata/B0FieldIdentifier.yaml

```diff
@@ -3,11 +3,20 @@ name: B0FieldIdentifier
 description: |
   The presence of this key states that this particular 3D or 4D image MAY be
   used for fieldmap estimation purposes.
-  The `B0FieldIdentifier` MUST be a unique string within one participant's tree,
-  shared only by the images meant to be used as inputs for the estimation of a
+  The `B0FieldIdentifier` MUST be a
+  [BIDS URI](/02-common-principles.html#bids-uri-pointing-to-files-within-and-outside-of-bids-datasets)
+  pointing to a file within one
+  participant's tree.
+  Using forward-slash separated paths is
+  [DEPRECATED](/02-common-principles.html#definitions).
+  The value of this field MUST be shared only by the images meant to be used
+  as inputs for the estimation of a
```

Comment:
This is a unique string, not a URI. It's just an identifier so that you can do things like:

```Python
fieldmap = layout.get(B0FieldIdentifier="myidentifier", extension=".nii.gz")
```